queries.sql

```diff
@@ -103,4 +103,77 @@ FROM owners o
 JOIN animals a ON o.id = a.owner_id
 GROUP BY o.full_name
 ORDER BY COUNT(a.id) DESC
+LIMIT 1;
+
+--1--
+SELECT a.name
+FROM visits v
+JOIN animals a ON a.id = v.animal_id
+JOIN vets vt ON vt.id = v.vet_id
+WHERE vt.name = 'William Tatcher'
+ORDER BY v.visit_date DESC
+LIMIT 1;
+
+--2--
+SELECT COUNT(DISTINCT v.animal_id)
+FROM visits v
+JOIN vets vt ON vt.id = v.vet_id
+WHERE vt.name = 'Stephanie Mendez';
+
```

Comment:
- [ ] The query 

```sql

SELECT COUNT(DISTINCT v.animal_id) FROM visits v JOIN vets vt ON vt.id = v.vet_id WHERE vt.name = 'Stephanie Mendez';
```

 is incorrect ❌ . 

The `COUNT(DISTINCT)` function should be used on the `animal_id `column, not the v column. For example the query can be written as following:

```sql

SELECT COUNT(DISTINCT a.id)
FROM visits v
JOIN animals a ON a.id = v.animal_id
JOIN vets vt ON vt.id = v.vet_id
WHERE vt.name = 'Stephanie Mendez';


```

This query will count the number of distinct animals that Stephanie Mendez has seen ✅ .  

The original query that you wrote would count the number of distinct visits that Stephanie Mendez has made, which is not what the question is asking for. However the question that the query is trying to answer is: How many different animals did Stephanie Mendez see?

Reply:
Hi @shella12, thank you so much for your review and feedback. I have adjusted the query as per your request.
![image](https://github.com/kifle23/vet-clinic-database/assets/68891198/a23b15ea-2e65-4761-84f5-0a4987653000)